crates/dids-v2/src/bearer_did.rs

```diff
@@ -0,0 +1,182 @@
+use crypto::key_manager::{local_key_manager::LocalKeyManager, KeyManager};
+
+use crate::{
+    did::DID,
+    document::{Document, VerificationMethodSelector},
+    portable_did::PortableDID,
+};
+
+#[derive(Debug)]
+pub struct BearerDID<T: KeyManager> {
+    pub did: DID,
+    pub key_manager: T,
+    pub document: Document,
+}
+
+pub fn from_portable_did(portable_did: PortableDID) -> Result<BearerDID<LocalKeyManager>, String> {
```

Comment:
torn on DX here, some places I do `DID::parse` but here it's a root level function

fine for now